cli/tests/e2e/test_dependency_aware_rules.py

```diff
@@ -293,3 +294,37 @@ def test_no_lockfiles(run_semgrep: RunSemgrep, monkeypatch, tmp_path, snapshot):
         ).as_snapshot(),
         "results.txt",
     )
+
+
+# These are file targets which are ecosystems that we can calculate the path to
+# transitivity for
+@pytest.mark.parametrize(
+    "target_supports_path_to_transitivity",
+    [
+        "targets/dependency_aware/log4j/maven_dep_tree.txt",
+    ],
+)
+def test_child_construction(
```

Comment:
I think I'd just move this into the `test_paths_to_transitivity` file, and maybe move that to the unit test folder.